Client/src/components/common/footer.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```

Comment:
remove unnecessary classes 
```suggestion
import React from 'react';
```